### DISCUSSION TITLE
Add support for pinned messages

### DISCUSSION THREAD

**Discussion by Kabouik** [d99kris/nchat#152] [Ideas]
It would be useful to support pinning messages (for oneself or for all) and showing already pinned messages, possibly with a `Go to` keybind.

**Comment by d99kris** [comment]
Hi @Kabouik,

Thanks for suggesting a new feature. I think it's something that maybe can be considered, but in the short term, I will probably not have time to look into this.

In any case, I encourage anyone else interested in this feature to upvote it. This will help gauge the level of interest.

**Comment by Disonantemus** [comment]
#168 is similar to this, but only the "show already pinned" part, I did vote that, as @d99kris suggested.

**Comment by d99kris** [comment]
In e47423e / `v5.15.23` support for pinned message indicator and jump to pinned message (ctrl-p) was added for Telegram and WhatsApp. (Underlying Signal library does not support getting message pinned info)

In 77b16f0 / `v5.15.24` support for pin/unpin of selected message (alt-p) was added for Telegram. (Underlying WhatsApp and Signal libraries do not support message pin/unpin)
